Gemfile

```diff
@@ -38,6 +38,9 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'pry-rails'
+  gem "rspec-rails", "~> 4.1.0"
+  gem 'database_cleaner'
+  gem 'rexml'
```

Comment:
Do we always need to expose these gems? Are they using in all the environments?

Reply:
No, they don't. Ok I check that